CHANGELOG.md

```diff
@@ -2,6 +2,38 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.0.3] 2019-12-13
+
+### Improvements
+
+- Add the command mode in dmctl
+- Support replicating the `AlTER DATABASE` DDL statement
+- Optimize the error message output
+
+### Bug fixes
+
+- Fix the panic-causing data race issue occurred when the full import unit pauses or exits
```

Comment:
a little strange? 🤔 the same with L31.

Reply:
Fix the panic issue during the import unit pauses or exits because of the data race?

Reply:
Is `during` suitable for `pauses` and `exists`? maybe you can ask for i18n team. 😢